app/views/helpers/category/update.phtml

```diff
@@ -36,6 +36,49 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="defaultOrder"><?= _t('sub.feed.preferred_order')?> </label>
+				<div class="group-controls">
+					<select name="defaultOrder" id="defaultOrderCategory" class="w50">
+						<option value="DESC" <?=
+							$this->category->defaultOrder() === 'DESC' ? 'selected="selected"' : ''  ?>><?=
+							_t('conf.reading.sort_order.newer_first') ?></option>
+						<option value= "ASC" <?=
+							$this->category->defaultOrder() === 'ASC' ? 'selected="selected"' : ''  ?>><?=
+							_t('conf.reading.sort_order.older_first') ?></option>
+					</select>
+				</div>
+			</div>
+
+			<div class="form-group">
+				<label class="group-name" for="defaultSort"><?= _t('sub.feed.preferred_sort')?> </label>
+				<div class="group-controls">
+					<select name="defaultSort" id="defaultSortCategory" class="w50">
+						<option value="id" <?=
```

Comment:
We still need to add a first blank option to use the default sort (i.e. clear the category-local preference)